packages/sdk/test/validations.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { voucherifyClient as client } from './client'
+import { generateVoucher } from './utils/generateVoucher'
+import { generateRandomString } from './utils/generateRandomString'
+import { generatePromotionTier } from './utils/generatePromotionTier'
+
+describe('Validations API', () => {
+	it('while validating not existing code, should get error message', async () => {
+		const response = await client.validations.validateVoucher(generateRandomString(55))
```

Comment:
try to apply exit early pattern + assertion for "valid"
Some idea (to confirm)

```ts
expect(response.valid).toBe(false)

if(response.valid){ // narrow down ts types, as far as I know, JestJs assertions can't do that right now
  return
}

expect(typeof response.reason).toBe('string')
expect(typeof response.error?.message).toBe('string')
expect(typeof response.error?.key).toBe('string')
expect(typeof response.error?.code).toBe('number')

```

the same for other tests